moto/ec2/responses/transit_gateways.py

```diff
@@ -46,20 +46,21 @@ def modify_transit_gateway(self):
     <transitGateway>
         <transitGatewayId>{{ transit_gateway.id }}</transitGatewayId>
         <ownerId>{{ transit_gateway.owner_id }}</ownerId>
-        <description>{{ transit_gateway.description }}</description>
+        <description>{{ transit_gateway.description if transit_gateway.description != None }}</description>
```

Comment:
Can you double-check if this syntax works properly. I vaguely remember seeing in other places that `None` checks in jinja templates are done like this: 
```
         <description>{{ transit_gateway.description if transit_gateway.description is not none }}</description>
```
Also, shouldn't there be an `else` clause as well?
```
         <description>{{ transit_gateway.description if transit_gateway.description is not none else '' }}</description>
```
or perhaps simply:
```
         <description>{{ transit_gateway.description or '' }}</description>
```
?

Reply:
Updated